webpack/envs/clientNovoDevelopmentConfig.js

```diff
@@ -97,11 +173,10 @@ export const clientNovoDevelopmentConfig = {
     new webpack.ProvidePlugin({
       $: "jquery",
       jQuery: "jquery",
-      "window.jQuery": "jquery",
       jade: "jade/runtime.js",
       waypoints: "jquery-waypoints/waypoints.js",
+      "window.jQuery": "jquery",
```

Comment:
Also wonder if we can drop jquery and add it back as needed when migrating apps to novo.

Reply:
Same as previous:

> Probably not, these weren't added in this PR and I have a task for cleaning up Webpack once it's stabilized. There is likely a lot that can be removed and deduplicated.